allyourpasswords/Views/EditViewController.swift

```diff
@@ -138,7 +138,7 @@ class EditViewController : NSViewController {
     @IBAction func cancelButtonClicked(_ sender: NSButton) {
         let container = self.parent as! ContainerViewController
         let rowCount = try! db?.scalar(login.table.count)
-
+        
```

Comment:
Oh yeah ok so this whitespace is just trailing, be sure in the future to delete the trailing whitespace, especially on blank lines

Reply:
Will do - thanks! I'll remove it.